misk/src/test/kotlin/misk/client/ClientMetricsInterceptorTest.kt

```diff
@@ -40,17 +40,19 @@ internal class ClientMetricsInterceptorTest {
   @MiskTestModule
   val module = TestModule()
 
-  @Inject private lateinit var jetty: JettyService
+  @Inject
+  private lateinit var jetty: JettyService
```

Comment:
can you make sure you have the square formatter installed in your intellij?

https://github.com/square/java-code-styles